tracing-core/src/metadata.rs

```diff
@@ -167,22 +163,9 @@ impl<'a> Metadata<'a> {
         self.target
     }
 
-    /// Returns the path to the Rust module where the span occurred, or
-    /// `None` if the module path is unknown.
-    pub fn module_path(&self) -> Option<&'a str> {
-        self.module_path
-    }
-
-    /// Returns the name of the source code file where the span
-    /// occurred, or `None` if the file is unknown
-    pub fn file(&self) -> Option<&'a str> {
-        self.file
-    }
-
-    /// Returns the line number in the source code file where the span
-    /// occurred, or `None` if the line number is unknown.
-    pub fn line(&self) -> Option<u32> {
-        self.line
```

Comment:
I think we should keep these functions as well, but have them call the appropriate method on the `Location` type.

Reply:
updated :)